app/views/defendants/_offences.html.haml

```diff
@@ -14,6 +14,8 @@
       %tr.govuk-table__row
         %td.govuk-table__cell
           = offence.title || t('generic.not_available')
+          %span.vcd-body-secondary
```

Comment:
i would be tempted to name this `app-body-secondary`. reason being that the apps name could change and currently VCD does not match the name of the repo and namespaces.